File: api/Entities/Admin/VisaTransaction.cs
namespace api.Entities.Admin
{
    public class VisaTransaction: BaseEntity
    {
        public int ApplicationNo { get; set; }
        public string CandidateName { get; set; }
        public int VisaItemId { get; set; }
        public string VisaNo { get; set; }
        public string VisaCategory { get; set; }
        public int CvRefId { get; set; }
        public int DepItemId { get; set; }
        public DateTime VisaAppSubmitted { get; set; }
        public DateTime VisaApproved { get; set; }
        public DateTime VisaDateG { get; set; }
        public int CustomerId { get; set; }
        
    }
}

File: client/src/app/_models/admin/visaTransaction.ts
export interface IVisaTransaction
{
    id: number;
    customerKnownAs: string;
    applicationNo: number;
    candidateName: string;
    cvRefId: number;
    visaItemId: number;
    visaNo: string;
    visaCategory: string;
    depItemId: number;
    visaAppSubmitted: Date;
    visaApproved: Date;
    visaDateG: Date;
    customerId: number;
}